wormhole-connect/src/utils/index.ts

```diff
@@ -408,6 +408,12 @@ export const isWrappedToken = (token: TokenConfig, chain: Chain) => {
   return token.nativeChain !== chain;
 };
 
+// Canonical tokens may be Wormhole-wrapped tokens that are canonical on the chain
+// e.g., Wormhole-wrapped Ethereum USDC is canonical on Sui
+export const isCanonicalToken = (token: TokenConfig, chain: Chain) => {
+  return token.key === 'USDCeth' && chain === 'Sui';
```

Comment:
@kev1n-peters Is canonical token concept only valid for Sui?

Reply:
It's a general concept, but this is the only token I know of now that needs to be special-cased like this

Reply:
We may need to expand this list in the future